### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by JuzzKidd**
Added win32 implementation for CryptoUtils::prng_seed

### CODE REVIEW

File: lib/Transforms/Obfuscation/CryptoUtils.cpp [hunk 1]
```diff
@@ -620,15 +624,33 @@ void CryptoUtils::populate_pool() {
...
 
 void CryptoUtils::prng_seed() {
+  LLVMContext &ctx = llvm::getGlobalContext();
+
+#if defined(_WIN32)
+  HCRYPTPROV hProvider = 0;
+
+  if (::CryptAcquireContextW(&hProvider, 0, 0, PROV_RSA_FULL, CRYPT_VERIFYCONTEXT | CRYPT_SILENT)) {
+    const DWORD dwLength = 8;
+    BYTE pbBuffer[dwLength] = {};
```
Comment (cryptopathe) [inline lib/Transforms/Obfuscation/CryptoUtils.cpp]: I don't think that defining the pbBuffer variable is necessary, right?

File: lib/Transforms/Obfuscation/CryptoUtils.cpp [hunk 2]
```diff
@@ -620,15 +624,33 @@ void CryptoUtils::populate_pool() {
 }
 
 void CryptoUtils::prng_seed() {
+  LLVMContext &ctx = llvm::getGlobalContext();
+
+#if defined(_WIN32)
+  HCRYPTPROV hProvider = 0;
+
+  if (::CryptAcquireContextW(&hProvider, 0, 0, PROV_RSA_FULL, CRYPT_VERIFYCONTEXT | CRYPT_SILENT)) {
+    const DWORD dwLength = 8;
```
Comment (cryptopathe) [inline lib/Transforms/Obfuscation/CryptoUtils.cpp]: The dwLength variable is not required, if I understand well.

**Comment by cryptopathe** [comment]
Thank you! I have made two comments with respects to lines 633 and 634. Can you review them?

**Comment by JuzzKidd** [comment]
Yes you are right. They must be from my test code. Fixed now!

Thank you for review!

**Pull request closed without merging**